### PR TITLE
Remove ManagedROS2Node

### PR DESCRIPTION
**Pull request by ds58**

### CODE REVIEW

**Review by calvertdw — APPROVED** [review]

**Comment by calvertdw** [comment]
I think it might be important that Luigi and Stephen sign off on this. I don't know how safety critical that would be.

File: ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/networkProcessor/modules/ToolboxModule.java [hunk 1]
```diff
@@ -65,9 +63,7 @@ public abstract class ToolboxModule implements CloseableAndDisposable
    protected final String robotName;
    protected final FullHumanoidRobotModel fullRobotModel;
 
-   private final boolean manageROS2Node;
```
Comment (TomaszTB) [inline ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/networkProcessor/modules/ToolboxModule.java:68]: This boolean seems unrelated to the ManagedROS2Node (despite its name). Should it really be removed?

Comment (ds58) [reply ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/networkProcessor/modules/ToolboxModule.java:68]: It didn't seem important to me. But, I've put it back just to reduce changes if they are unnecessary.

**Review by TomaszTB — APPROVED** [review]
Seems good to me, but I'm not an expert on this code
